src/constants/featureFlags.ts

```diff
@@ -1,3 +1,4 @@
 export const FEATURE_FLAGS = {
   GGS_WHITELISTED_REPOS: "ggs_whitelisted_repos",
+  REDUCE_BUILD_TIMES_WHITELISTED_REPOS: "reduce_build_times_whitelisted_repos",
```

Comment:
for feature flags going forward we prefer to use something like `isBuildTimeFlowWhitelisted` or `isQuickieWhitelisted`

Reply:
4be5c4826360d1592e35d47855f5c93ca0901053